### DISCUSSION TITLE
Suggestion: Test many PCIe Video Capture Cards (HDMI/DVI/SDI)

### DISCUSSION THREAD

**Discussion by BegoneRaven** [geerlingguy/raspberry-pi-pcie-devices#189] [Ideas]
Realized that many capture cards are PCIe x1, and hence this suggestion :) 
Magewell capture cards proudly [claim](https://www.magewell.com/kb/000020047/detail):
> Supported OS: ARM
> Recommended OS: (Not able to see ARM though)

**Comment by 6by9** [comment]
It looks like it's additional out-of-tree drivers - https://github.com/jat001/magewell-pro-capture
